examples/mock/mockoon-paymentsbbvoucher.json

```diff
@@ -411,6 +411,418 @@
       ],
       "enabled": true,
       "responseMode": null
+    },
+    {
+      "uuid": "236f74d1-94d9-4a39-9d0c-17fb74e68302",
+      "documentation": "Request for voucher redemption by sending the voucher number, the merchant name and merchant payment details.",
+      "method": "post",
+      "endpoint": "vouchers/voucher_redeemption",
+      "responses": [
+        {
+          "uuid": "525fc27b-5ec1-41b2-b2ad-163b5352228e",
+          "body": "{\n  \"result_status\": \"Voucher is successfully redeemed\"\n}",
+          "latency": 0,
+          "statusCode": 200,
+          "label": "Voucher is successfully redeemed",
+          "headers": [],
+          "bodyType": "INLINE",
+          "filePath": "",
+          "databucketID": "",
+          "sendFileAsBody": false,
+          "rules": [
+            {
+              "target": "body",
+              "modifier": "voucher_number",
+              "value": "^((?!(0))[0-9]{0,19})",
```

Comment:
Probably missing $ at the end of the regex in mockoon

Reply:
![image](https://user-images.githubusercontent.com/119664031/212356029-09fb7187-575d-436d-a17f-56cae3c0e5f2.png)

Reply:
Nice catch! Thank you ;)